.github/workflows/yaml-lint.yaml

```diff
@@ -5,27 +5,20 @@ on:
       - "**.yaml"
 
 jobs:
-  yaml-linter:
+  test:
```

Comment:
still we are doping yaml-linter do we need to rename it

Reply:
done.